### ISSUE TITLE
0.3 release

### ISSUE THREAD

**Issue by benorama** [kenliu/grails-elastic-beanstalk#29]
Hello,

Do you have any plan for 0.3 release?
There is quite a lot of changes since 0.2.

Thanks.

**Comment by kenliu** [comment]
yes. In fact, I was planning on releasing it some time this week.

**Comment by benorama** [comment]
Good news, thanks.

**Comment by kenliu** [comment]
BTW do you work with @jvdrean?

**Comment by benorama** [comment]
Yes indeed ;)
We have integrated your plugin in our build pipeline (Travis-based deployment to AWS Beanstalk).
Right now, we used a forked version on our own maven repo, but the goal would be to use 0.3 version.

**Comment by kenliu** [comment]
excellent! would you guys be interested in becoming committers on the
project?

**Comment by benorama** [comment]
Sure, why not.
Committers or pull requests, as you want.

**Comment by benorama** [comment]
Any plan to push 0.3 version to Grails.org? ;)

We are using a SNAPSHOT version of latest master commit and it is working great for us.

**Comment by mortizbey** [comment]
Grails.org still keeps v0.2. Is 0.3 available? Is this plugin still mantained? Its a great plugin!

**Comment by kenliu** [comment]
Hey guys -

I will make a best effort to release 0.3 by the end of this month.

**Comment by mortizbey** [comment]
Thanks!! That would be nice!

**Comment by klemensz** [comment]
+1

**Comment by neoecos** [comment]
Hi there, any help need ?

I'been seeing very big diferences from the snapshot and the code in this repo. Please push to 0.3 final.

**Comment by kenliu** [comment]
Yes, sorry I have not been taking care of this. Sorry to keep you all waiting. I am working on transitioning maintenance/releases of this plugin to someone who is better able to move this forward.

**Comment by neoecos** [comment]
@kenliu I think there is a lot of work done for the 0.3 release . I will be glad to help with anything you need in order to release the 0.3. 

Please let know.

**Comment by benorama** [comment]
Indeed, we are using the plugin for our deployment, a 0.3 version of it without any issue.
I suppose that you could publish it to grails.org, what’s left to be done?

**Comment by neoecos** [comment]
I'm using it, because @benorama talk at gr8 last year =)

**Comment by benorama** [comment]
Hi guys,

FYI, we are starting to use a simple shell script for our deployments, based on AWS CLI.
It replaces this plugin.

Here it is:
https://gist.github.com/benorama/b26e7a94fe80e8c54daf

**Comment by klemensz** [comment]
Can you push 0.3 to grailsCentral?
I get this error message: Error Could not find artifact org.grails.plugins:aws-elastic-beanstalk:zip:0.3 in grailsCentral (http://repo.grails.org/grails/plugins)
